openassessment/templates/openassessmentblock/oa_uploaded_file.html

```diff
@@ -5,7 +5,11 @@
     {% if header %}
         <header class="{{ class_prefix }}__display__header">
             <h3 class="{{ class_prefix }}__display__title">
-                {% trans header %}
+                {% if header == "yours" %}
+                    {% trans "Your Uploaded File" %}
+                {% else %}
+                    {% trans "Associated File" %}
+                {% endif %}
```

Comment:
How does everyone feel about the 2 approaches for dealing with translated parameters?

Passing a "flag" to the final template reduces duplicated strings and keeps the values to be translated near where they're used.
* Example call: https://github.com/edx/edx-ora2/blob/efischer/i18n_fixes/openassessment/templates/openassessmentblock/grade/oa_grade_complete.html#L34
* Example display: the lines this comment is attached to.

On the other hand, setting a `translated_*` variable in context and displaying that directly in the final template allows the caller to control the string exactly, and avoids any issues with the enumeration growing too large, or too slowly (as seen in )
* Example call: above linked file, lines 31 and 32
* Example display: https://github.com/edx/edx-ora2/blob/efischer/i18n_fixes/openassessment/templates/openassessmentblock/oa_submission_answer.html#L13

I can see the benefits and drawbacks of both, but I feel we should pick one way and stick with it.